backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/utils/Modifier.kt

```diff
@@ -86,3 +89,7 @@ internal fun Modifier.offsetWithSize(offset: IntrinsicMeasureScope.(size: IntSiz
 internal fun Modifier.alignBy(anchor: Offset, alignment: Alignment): Modifier = offsetWithSize { size ->
     anchor.round() - alignment.align(IntSize.Zero, size, layoutDirection)
 }
+
+@OptIn(ExperimentalComposeUiApi::class)
+internal fun Modifier.invisibleSemantic(): Modifier =
```

Comment:
it might be worth making this public - we have a similar function we're using in hotels